src/components/WwwFrame/TheHeader.vue

```diff
@@ -409,7 +411,7 @@ export default {
 		PromoBannerLarge,
 		PromoBannerSmall,
 		SearchBar,
-		TheLendMenu: () => import('@/components/WwwFrame/LendMenu/TheLendMenu'),
```

Comment:
This dynamic import was breaking the exp wrapper component, which has a duplicate slot with the lend menu

Reply:
So now that will get bundled into the header bundle instead of being an on demand import?

Reply:
I believe thats true, but I am not sure what repercussions that will have. @emuvente might know more?

Reply:
Yeah, that's all that should happen. We previously used that dynamic import to delay the fetching of lend menu data, but that's handled by kv-dropdown events now